src/editor/action-executor.ts

```diff
@@ -36,6 +39,67 @@ export class ActionExecutor {
         let preventDefaultEvent = true;
 
         switch (action.type) {
+            case EditActionType.InsertElseStatement: {
+                const newStatement = new ElseStatement(action.data.hasCondition);
+
+                if (action.data.outside) {
+                    // when the else is being inserted outside
+                    const elseRoot = context.lineStatement.rootNode as Module | Statement;
+                    newStatement.rootNode = elseRoot;
+                    newStatement.indexInRoot = context.lineStatement.indexInRoot;
+                    newStatement.body.push(new EmptyLineStmt(newStatement, 0));
```

Comment:
This causes module.ts:912 to throw an error when trying to insert an else at the cursor position below.

```
if ---:
elif ---:
    |
else:
```

It does not allow the insertion though, which is good.

Reply:
hmmm. I think it should not allow you to insert an `elif` or an `else` right after an `if`. 
What do you think? should we allow that? body-less if and elif statements? I'd personally feel like it will be more consistent with the rest of the tool + not having an empty statement there (that should be filled) is a syntax error.

Reply:
fixed in 2006117c1cb53d81040bd3240f33f752c94e7019

Reply:
Sorry, Github notifications were delayed for me so I only saw this right now.

Reply:
Yeah it should not allow it. I was just talking about the fact that it triggers some error in the console when you try.